test/e2e/baseline_test.go

```diff
@@ -819,7 +819,50 @@ func TestL4Telemetry(t *testing.T) {
 	})
 }
 
-func buildL4Query(src, dst echo.Instance) prometheus.Query {
+func TestLongConnL4Telemetry(t *testing.T) {
+	framework.NewTest(t).Run(func(tc framework.TestContext) {
+		for _, src := range apps.EnrolledToKmesh {
+			for _, dst := range apps.EnrolledToKmesh {
+				tc.NewSubTestf("from %q to %q", src.Config().Service, dst.Config().Service).Run(func(stc framework.TestContext) {
+					localDst := dst
+					localSrc := src
+					opt := echo.CallOptions{
+						Port:                    echo.Port{Name: "tcp"},
+						Scheme:                  scheme.TCP,
+						Count:                   10,
+						Timeout:                 time.Second,
+						Check:                   check.OK(),
+						To:                      localDst,
+						NewConnectionPerRequest: false,
+					}
+
+					query := buildL4Query(localSrc, localDst, "kmesh_tcp_sent_bytes_total")
+					stc.Logf("prometheus query: %#v", query)
+					err := retry.Until(func() bool {
+						stc.Logf("sending call from %q to %q", deployName(localSrc), localDst.Config().Service)
+						localSrc.CallOrFail(stc, opt)
```

Comment:
In addition this test wants to get metrics during the long connection process? However this function call is synchronous, and when it returns, the connection has been terminated.